en/admin/migration.inc

```diff
@@ -210,7 +244,7 @@ The following table shows [options] available with the **cubrid loaddb** utility
 
     This option commits periodically every time COUNT records are entered into the database. If this option is not specified, all records included in demodb_objects are loaded to the database before the transaction is committed. If this option is used together with the **-s** or **-i** option, commit is performed periodically every time 100 DDL statements are loaded. 
```

Comment:
We should change this:
If this option is not specified, all records included in demodb_objects are loaded to the database before the transaction is committed.

it is also better to explicitly mention the default behavior change. 
10.1 or older versions does not periodically commit without the option. ... To have the same behavior, you may specify a big number.